contents/docs/surveys/troubleshooting.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Troubleshooting and FAQs
+---
+
+Having trouble with surveys? Below are some tips for getting past some common issues
+
+The most common solution is...
+
+#### Update posthog-js
+
+Surveys requires either the latest version of the [PostHog snippet](/docs/libraries/js#option-1-add-the-javascript-snippet-to-your-html) or version 1.81.3 or greater of `posthog-js` to be installed on your website.
+
+We're always making improvements to the surveys feature, so you'll want to make sure that you're running the latest version.
+
+To check the version that you're using, you can run the following in your browser console:
+
+```js
+window.posthog.LIB_VERSION
+```
+
+## Surveys still not showing up
+
+There are a few reasons why your survey might not be showing up:
+
+### 1. Configuration is off
+
+Surveys are enabled through the surveys opt in toggle in the [app](https://app.posthog.com/surveys). If you're not seeing any surveys, make sure that this toggle is enabled.
+
+You can read more about [posthog-js configurations here](/docs/integrate/client/js#config).
+
+### 2. Content security policy
```

Comment:
Can you double check some of these points @neilkakkar? 🤗  and if you know more off the top of your head, feel free to add

Reply:
No these seem all good to me 👍